From 724cc5b8aa05dedecbc4b2a3082815a3eb093637 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Tue, 18 Nov 2014 13:54:28 -0500
Subject: [redhat] kabi: Adding symbol dev_printk (drivers/base/core.c) DRIVER CORE, KOBJECTS, DEBUGFS AND SYSFS

Message-id: <2c6c156db5c397b73f0dbf2a8bd6b6680cd4d9d4.1416318767.git.poros@redhat.com>
Patchwork-id: 100142
O-Subject: [PATCH RHEL7.1 23/63] kabi: Adding symbol dev_printk (drivers/base/core.c) DRIVER CORE, KOBJECTS, DEBUGFS AND SYSFS
Bugzilla: 1070274
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1070274
    https://bugzilla.redhat.com/show_bug.cgi?id=1070274

    Description:
    ============
    Log a message for a specific device; one of the most basic things to do.
    Symbol used in drbd

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7.0 45/80] kabi: Adding symbol dev_printk (drivers/base/core.c)
 Jiri Olsa <jolsa@redhat.com> (Tue,  4 Mar 2014 16:31:45 +0100)
 Description:
 ============
 Log a message for a specific device; one of the most basic things to do.

 Reviews:
 ========
 Re: [PATCH RHEL7.0 45/80] kabi: Adding symbol dev_printk (drivers/base/core.c)
 Prarit Bhargava <prarit@redhat.com> (Wed, 05 Mar 2014 07:54:15 -0500)
 Acked-by: Prarit Bhargava <prarit@redhat.com>
 P.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/dev_printk
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/dev_printk
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/dev_printk

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/dev_printk b/redhat/kabi/kabi-module/kabi_ppc64/dev_printk
new file mode 100644
index 0000000..6d732ee
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/dev_printk
@@ -0,0 +1,2 @@
+#1-
+0x186823ad dev_printk vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/dev_printk b/redhat/kabi/kabi-module/kabi_s390x/dev_printk
new file mode 100644
index 0000000..f5f3e4a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/dev_printk
@@ -0,0 +1,2 @@
+#1-
+0x161fb4bf dev_printk vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/dev_printk b/redhat/kabi/kabi-module/kabi_x86_64/dev_printk
new file mode 100644
index 0000000..58723a2
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/dev_printk
@@ -0,0 +1,2 @@
+#1-
+0x18abe2e5 dev_printk vmlinux EXPORT_SYMBOL
-- 
1.7.1